ivoabib.bib

```diff
@@ -1090,3 +1099,34 @@ @ARTICLE{2015A+A...574A..36R
   adsnote = {Provided by the SAO/NASA Astrophysics Data System}
 }
 
+@Misc{note:cooluris,
+	year=2008,
+	month=Mar,
+	url={https://www.w3.org/TR/cooluris/},
+	title={Cool URIs for the semantic web},
+	author={Leo Sauermann and Richard Cyganiak},
+	howpublished={{W3C Interest Group Note}}
+}
+
+@Misc{note:rdfprimer,
+	year=2014,
+	month=Jun,
+	url={https://www.w3.org/TR/rdf11-primer/},
+	title={RDF 1.1 Primer},
+	author={Guus Schreiber and Yves Raimond},
+	howpublished={{W3C Interest Group Note}}
+}
+
+@INPROCEEDINGS{2005ASPC..347...29T,
+   author = {{Taylor}, M.~B.},
+    title = "{TOPCAT {\amp} STIL: Starlink Table/VOTable Processing Software}",
```

Comment:
I'm not a LaTex expert, so this might not be important but it looks inconsistent. Some titles are wrapped in double quotes, some are wrapped in curly brackets {}, this title is in both, but the keywords TOPCAT and STIL are not wrapped in curly brackets {} ?